_posts/2015-06-07-cosmic-neutrinos.md

```diff
@@ -0,0 +1,45 @@
+---
+layout: post
+title: Improved limits in search for ultrahigh-energy cosmic neutrinos
+author: RaoOfPhysics
+---
+
+_Pierre Auger Observatory finds no neutrino candidates with energies of greater than 1 EeV in data collected between Jan 2004 and Jun 2013_
+
+We are all bathed in a constant stream of particles knows as cosmic rays[^1], which are produced mostly outside the Solar System.
+These high-energy particles were initially suspected to have Earthly origins, but Victor Hess demonstrated in 1912 that they arrive on our planet from non-terrestrial sources.
+The enigmatic nature of cosmic rays has captivated scientists for decades.
+
+In 1966, Kenneth Greisen as well as Georgiy Zatsepin and Vadim Kuzmin calculated what has since been known as the [Greisen–Zatsepin–Kuzmin (GZK) limit](https://en.wikipedia.org/wiki/Greisen%E2%80%93Zatsepin%E2%80%93Kuzmin_limit), which is an upper bound for the energies of cosmic rays originating from very distant sources, based on extrapolations from lower energies.
```

Comment:
In 1966, scientists proposed that the energy of cosmic rays originating from very distant sources can't exceed a certain bound. This G-Z-K limit comes from energy losses when UHECRs interact with photons from the cosmic microwave background.